spec/helper/MockHelper.ts

```diff
@@ -422,7 +422,7 @@ class PlayerEventHelper implements EventEmitter {
     });
   }
 
-  fireVideoPlaybackQualityChangedEvent(bitrate: number): void {
+  fireVideoPlaybackQualityChangedEvent(bitrate: number, frameRate: number): void {
```

Comment:
Please make `frameRate` optional, so we can test events with and without `frameRate` being available.